src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,72 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+import { FC } from 'react';
+import { TodoStatusOptions } from '../../enums/TodoStatusOptions';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+interface Props {
+  searchQuery: string;
+  setSearchQuery: (searchQuery: string) => void;
+  selectedOption: string;
+  setSelectedOption: (selectedOption: string) => void;
+}
 
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button
-          data-cy="clearSearchButton"
-          type="button"
-          className="delete"
+export const TodoFilter: FC<Props> = ({
+  searchQuery,
+  setSearchQuery,
+  selectedOption,
+  setSelectedOption,
+}) => {
+  const handleSearch = (event: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
You used this function in the `change` event. Soin name you show that. 

`handleChange`, `onChange`

```suggestion
  const handleChangeSearch = (event: React.ChangeEvent<HTMLInputElement>) => {
```